app/styleguide/styleguide.scss

```diff
@@ -0,0 +1,75 @@
+
+#sgLayout {
+  display: flex;
+  flex-direction: row;
+}
+
+
+#sgNav, #sgContent {
+  padding: 0.5rem 1rem;
+}
+
+#sgNav {
+  display: flex;
+  flex-direction: column;
+  border-right: 2px solid #e3e3e3;
+
+  a {
+    display: block;
+    padding: 0.5em 1em;
+  }
+}
+
+#sgContent {
+  & > div {
+    margin-bottom: 3em;
+    max-width: 35em;
+  }
+}
+
+.sg-example {
+  margin-bottom: 2rem;
+}
+
+.sg-button-list > * {
+  margin: 0.5em;
+}
+
+.brand-color-list {
+  > * {
+    margin-bottom: 1em;
+  }
+}
+
+ul.sg-spaced {
+  li {
+    margin-bottom: 1rem;
+  }
+}
+
+dd {
+  margin-top: var(--brand-space1);
+  margin-bottom: var(--brand-space2);
+}
+
+.brand-color {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+
+  position: relative;
+  --_color: gray;
+  --_border-color: gray;
+
+  &:before {
+    content: "";
+    display: block;
+    position: relative;
+    margin-right: var(--brand-space2);
+    width: 75px;
+    height: 30px;
+    border-radius: 3px;
+    border: 1px solid var(--_border-color);
+    background-color: var(--_color);
+  }
+}
```

Comment:
Yeah lets commit with the inaccessible "dummy colours" for now, and we can fix all that thing when start to add actual "design" to the style guide. 

We will make sure the future iterations the colours are fully accessible

Reply:
My apologies for the confusion here @jupiter007 , the colours we have in the styleguide right now are not meant to be the final colours. My understanding is that we will only have the final colours around Aug/Sept.

My apologies for not making it clear that these are only temporary colours so that we can just get on with building :sweat_smile:

Reply:
Thanks @andrewebdev. Yea, I realize that these are not final colors, but I thought I would just put in a reminder here that the final colors be checked for sufficient contrast.